### PR TITLE
[WIP] Add test functions for issues related to loss-of-precision

### PR DESCRIPTION
**Pull request by NimaSarajpoor**

### CODE REVIEW

File: tests/test_precision.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import stumpy
+from stumpy import config
+
+
+def test_mpdist_snippets_s():
+    # This test function raises an error if the distance between
+    # a subsequence (of length `s`) and itelf becomes non-zero
+    # in the performant version. Fixing this loss-of-precision can
+    # result in this test being passed.
```
[the remark this answers is not in the document]
Comment (seanlaw) [reply tests/test_precision.py:13]: Yep. In the future, we would simply write the unit test FIRST and push it as a commit, which causes a failed test. Then, in a separate commit we update the code (not unit test) to pass the precision test (and all other tests). This is the core of test-driven development

[the remark this answers is not in the document]
Comment (NimaSarajpoor) [reply tests/test_precision.py:13]: Cool. Thank you for the advice :)

**Comment by seanlaw** [comment]
Good to merge?

**Comment by NimaSarajpoor** [comment]
Sure. I think it is good.

**Comment by seanlaw** [comment]
Thanks @NimaSarajpoor!

**Pull request merged**